cli/src/commands/device/export_recovery_device.rs

```diff
@@ -2,14 +2,14 @@
 
 use std::path::PathBuf;
 
-use libparsec::save_recovery_device;
+use libparsec::{DateTime, DeviceLabel};
 
 use crate::utils::*;
 
 crate::clap_parser_with_shared_opts_builder!(
     #[with = config_dir, device, password_stdin]
     pub struct Args {
-        /// Recovery device output
+        /// Path where to save recovery device data
         #[arg(short, long)]
```

Comment:
```suggestion
        #[arg]
```

I would say to directly pass the output path as an arg over an option